docs/pom.xml

```diff
@@ -70,8 +70,27 @@ under the License.
     <!-- modules with documentation not need to install and deploy -->
     <maven.install.skip>true</maven.install.skip>
     <maven.deploy.skip>true</maven.deploy.skip>
+
+    <downloadDirectory>pom</downloadDirectory>
+    <downloadArtifactId>maven-parent</downloadArtifactId>
   </properties>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-remote-resources-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>site-download</id>
+            <phase>pre-site</phase>
+            <inherited>false</inherited>
```

Comment:
I don't think anyone inherits from the site-pom anyways....

Reply:
`docs/pom.xml` it is a parent pom for other docs site - without it it will be executed in all child

Reply:
Sorry, wasn't aware. Then obviously this is necessary.